spec/controllers/grade_entry_forms_controller_spec.rb

```diff
@@ -149,12 +149,13 @@
     end
 
     it 'ignores the total column if given in the csv file' do
+      # show_total needs to be on when uploading a csv file with the "Total" column included
+      grade_entry_form_with_data.update(show_total: true)
       post :upload, params: { id: grade_entry_form_with_data.id, upload_file: @file_total_included, overwrite: true }
       expect(response.status).to eq(302)
       expect(flash[:error]).to be_nil
       expect(response).to redirect_to(grades_grade_entry_form_path(grade_entry_form_with_data))
-
-      # Check that the total column is the actual total and not the incorrect total given in the file
+      # Check that the total column is the actual total and not the incorrect  v
```

Comment:
Please revert these changes: include the blank line, and restore the previous comment, which I think you accidentally modified.

Reply:
yup sorry, I meant to fix that before pushing silly mistake.